sdk/metric/aggregator/histogram/histogram.go

```diff
@@ -41,6 +41,19 @@ type (
 		state      state
 	}
 
+	// Config describes how the histogram is aggregated.
+	Config struct {
```

Comment:
No need to be exported (public)

Reply:
Our style guide is possible inconsistent on when to make the config struct, config struct fields, and apply methods exported. In this case I agree it's not a good idea to export. Done.